app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates :okta_id, uniqueness: true, presence: true
```

Comment:
do we not get a guid back in the OIDC id_token? we are tracking guids within Okta for users, and my understanding is that is still the preferred mechanism of connecting users via different systems.

Reply:
We do get ssoguid back, I wasn't sure if that was a requirement for this or what the plan is for guids long term. if we're planning to migrate to okta user ids or something.

Do you think I should save the sso guid during login? are new okta users (outside cru) getting new ssoguids?

Reply:
new accounts get guids created for them, I think we are staying with guids because it's so integral to every system we have, and there isn't a point in migrating away from them.

the dev1 environment isn't set up to generate guids currently though, mainly because it doesn't have any of our infrastructure attached to it.

Reply:
I would say we would want to record guids to make it easier to integrate with other systems in the future if needed

Reply:
If that is the case then I should be able to change to sso guid.

Reply:
changes made in [`4f309af` (#469)](https://github.com/CruGlobal/mobile-content-api/pull/469/commits/4f309af7dbbba695e56328faa68af6e1de24a453)